tests/features/custom_page/add_custom_page.feature

```diff
@@ -41,3 +46,27 @@ Feature: "Add custom page" visibility options.
     # Check that the collection content such as the 'Join collection block' is
     # available in context of the custom page.
     Then I should see the link "Leave this collection"
+
+    # I should not be able to add a custom page to a different collection
+    When I go to the homepage of the "Code Camp" collection
+    Then I should not see the link "Add custom page"
```

Comment:
This is a description of a business objective. A facilitator should only be able to add custom pages to the collection they are a facilitator of, not to other collections.